src/components/ebay-fake-menu/index.marko

```diff
@@ -34,37 +34,41 @@ $ var baseClass = input.classPrefix || "fake-menu";
         input.class
     ]
     style=input.style>
-    <div
-        class=`${baseClass}__items`
+    <ul
+        class=`fake-menu__items`
```

Comment:
Is this change intentional?

Reply:
It is intentional, the styling looked good after using it. The issue is that with the fake-menu-button__items class, the bullet points were showing and the margins were off. 

Shifting to the fake-menu__items class fixed everything, so I figured we could just hard code it.

Reply:
Is that a bad idea? I wasn’t 100% sure but it looked great so I thought it was a good direction to go

Reply:
Hmm. This suggest that we might need a skin change also. Let me check...

Reply:
I don't see any issue with skin.